src/index.js

```diff
@@ -0,0 +1 @@
+console.log("Hello World!");
```

Comment:
- You should remove all console.log For best practice

Reply:
Yes @girma336, sure. Thank you. I have made the necessary changes.
Kindly review again.